md/installation_guide.md

```diff
@@ -0,0 +1,238 @@
+:::
+
+
+### Linux / OSX
+
+Elasticsearch, even in a Docker container, requires specific configuration of virtual memory. see [elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html)
+
+```
+# configure runtime environnement
+sudo sysctl -w vm.max_map_count=262144
+```
+or edit file `/etc/sysctl.conf` file
+
+
+### Interactive installer 
+
+Run the product using `bonita-ici` (`bonita-ici.bat` on Windows platform)script that is located inside the `bin` folder
+
+It will deploy an elasticsearch installation and configure and deploy the ICI application.
+Then it will deploy living application configured to use this ICI installation.
+
+Example:
+
+```
+./bonita-ici -d jdbc:postgresql://192.23.1.25:5432/bonita\
+            -u bonita \
+            -p bpm \ 
+            -b https://192.23.1.24:8080/bonita \
+            -U install \
+            -P install \
+            -H 192.23.1.23
+```
+
+Parameters are:
+
+* d: the jdbc url of the bonita datasource to use
+* u: the username for the datasource
+* p: the username for the datasource
+* b: the url of bonita platform
+* U: the username to use when deploying LAs on bonita (default 'install')
+* P: the password to use when deploying LAs on bonita (default 'install')
+* H: the host on which this current host is accessible form bonita (i.e. the external ip),  (default 'localhost')
+
+Ports on which Elasticsearch and ICI application will be deployed can be customized
+* i: the port of the deployed ICI application (default '8082') 
+* e: the port of the deployed elasticsearch (default '9200')    
+
+:::note
+use `./bonita-ici --help` displays all options
+:::
+
+It can be stopped using `bonita-ici stop`
+
+## Production mode 
+The deliverable is composed of one standalone application, two Bonita Living Applications, and an installer that will
+help you install the backend and deploy the two living applications in your current instance of the Bonita platform.
+
+```
+bonita-ici-<VERSION>.zip
+    |---- ici-application-<VERSION>.zip           // Standalone application for A.I.
+    |---- la-configuration-<VERSION>.zip          // Bonita L.A. for I.C.I. configuration
+    |---- la-operations-management-<VERSION>.zip  // Bonita L.A. for Operations Management
+    |---- la.properties                           // properties used by L.A. deployer utility
+    |---- installation-guide.md                   // this file
+    |---- bin/bonita-ici                          // an utility that start the dockers and deploy L.A.s
+    |---- bin/bonita-ici.bat                      // an utility that start the dockers and deploy L.A.s (Windows)
+    |---- jdbc_drivers/                           // the folder where to put the jdbc driver of the Bonita platform database
+```
+
+
+### Elasticsearch
+
+ICI server requires elasticsearch. Supported version is 6.2.3 or above, in cluster or single node mode. 
```

Comment:
The ICI server requires an Elasticsearch database (not sure, but it seems to be missing something after Elasticsearch).
Supported versions are